base/files/file.h

```diff
@@ -420,6 +420,7 @@ class BASE_EXPORT File {
   bool async_ = false;
 
 #if defined(STARBOARD)
+  bool delete_on_close_;
```

Comment:
I assume other platforms support the DELETE_ON_CLOSE flag, why is it that we need a new member to be able to do it while they don't?